koku/reporting/provider/ocp/models.py

```diff
@@ -767,3 +767,166 @@ class Meta:
         decimal_places=6,
         null=True
     )
+
+class OCPStorageLineItem(models.Model):
+    """Raw report storage data for OpenShift pods."""
+
+    class Meta:
+        """Meta for OCPStorageLineItem."""
+
+        unique_together = ('report', 'namespace', 'persistentvolumeclaim',)
+
+    id = models.BigAutoField(primary_key=True)
+
+    report_period = models.ForeignKey('OCPUsageReportPeriod',
```

Comment:
This may come up when you start working on the report processor, but we might next want to add a column to report_period for report_type so we can track usage or storage separately. There are columns right now for when data summarization occurred, and since the summarization will be slightly different it might be nice to have separate entries. -- Still not needed for this PR.